src/main/java/com/rockset/jdbc/RocksetResultSet.java

```diff
@@ -1389,7 +1397,9 @@ private JsonNode column(int index) throws SQLException {
     try {
       ObjectMapper mapper = new ObjectMapper();
       Object onedoc = resultSet.get(rowIndex.get());
-      JsonNode docRootNode = mapper.readTree(mapper.writeValueAsString(onedoc));
+
+      JsonNode docRootNode = cachedDoubleParseResult.getOrCompute(onedoc, parseFunction(mapper));
```

Comment:
`docRootNode` can be computed and cached whenever `rowIndex` is incremented in [next](https://github.com/rockset/rockset-java-client/blob/5a7591f31984398bc95de30e180d704c60a3aec0/src/main/java/com/rockset/jdbc/RocksetResultSet.java#L234) method or when it is [reset](https://github.com/rockset/rockset-java-client/blob/5a7591f31984398bc95de30e180d704c60a3aec0/src/main/java/com/rockset/jdbc/RocksetResultSet.java#L220). Then the cached value can be used here to get the value for the requested column index